src/cljs/athens/db.cljs

```diff
@@ -417,17 +413,16 @@
   ([query n ignore-dup]
    (if (string/blank? query)
      (vector)
-     (let [results (->> (d/q '[:find [(pull ?node [:db/id :node/title :block/uid]) ...]
-                               :in $ ?query-pattern ?query
-                               :where
-                               [?node :node/title ?title]
-                               [(re-find ?query-pattern ?title)]
-                               [(not= ?title ?query)]]                ;; ignore exact match to avoid duplicate
-                             @dsdb
-                             (re-case-insensitive query)
-                             (when ignore-dup query))
-                        (take n))]
-       results))))
+     (let [ignore-dup             (when ignore-dup query)
```

Comment:
ignore-dup -> ignore-dup?

also using ignore-dup in filter is making it ambiguous. 
try a better name - query-ignore or better

Reply:
Noted. I'll update.